tests/fault_tolerance/deploy/test_deployment.py

```
@@ -0,0 +1,172 @@
+            )
+        )
+        procs[-1].start()
+    yield procs
+
+    for proc in procs:
+        logger.debug(f"{proc} waiting for join")
+        proc.join()
+        logger.debug(f"{proc} joined")
+
+
+def _inject_failures(failures, logger, deployment: ManagedDeployment):  # noqa: F811
```

Ruff (0.13.1)

Unused `noqa` directive (unused: `F811`)

Remove unused `noqa` directive

(RUF100)